charts/rancher-csp-adapter/0.1.0+up0.1.0-rc6/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Rancher CSP Adapter
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.22.0'
```

Comment:
Can you make this a closed range and follow this format? `'>= 1.22.0-0 < 1.XX.0-0'` where XX is the max Kubernetes version the chart supports plus 1.

Reply:
Never mind, I'm told this will eventually have to support psps so I'll have to upper bound this at 1.25

Reply:
Fixed this - let me know if this is not what you were thinking.